### PR TITLE
adding czech republic national sanctions list

### PR DESCRIPTION
**Pull request by bgmello**
Fixes opensanctions/crawler-planning#158

### CODE REVIEW

**Review by pudo — APPROVED** [review]
Looks good to me. We need some library tooling for the months thing soon! :)

File: datasets/cz/national_sanctions/crawler.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    )
+    if res:
+        countries = cast("List[str]", res.names)
+    else:
+        context.log.warning("Country not identified", text=res)
+        countries = None
+
+    if first_name is None:
+        entity = context.make("LegalEntity")
+        entity.id = context.make_slug(last_name)
```
Comment (pudo) [inline datasets/cz/national_sanctions/crawler.py:83]: We do need a more detailed ID here. Do we have any sort of extra key material we can use?

Comment (bgmello) [reply datasets/cz/national_sanctions/crawler.py:83]: For Legal entities, the only other information that is not related to the sanction is the country, we can use that. What do you think?

**Comment by bgmello** [comment]
I was impressed that apparently the python `datetime` library doesn't have any way to deal with months in other languages.

**Pull request merged**